.changeset/twelve-jokes-confess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@axelar-network/axelar-cgp-sui': minor
+---
+
+add previous signer retention in gateway setup
```

Comment:
can you accumulate changes from the following prs to the changeset file as well?

- https://github.com/axelarnetwork/axelar-cgp-sui/pull/73
- https://github.com/axelarnetwork/axelar-cgp-sui/pull/70

So, the release action will include those changes into the changelog file as well.